GalaxyZooTouchTable/GalaxyZooTouchTable/Models/TableUserFactory.cs

```diff
@@ -15,12 +15,12 @@ enum UserType
         Heart
     }
 
-    public class TableUser : INotifyPropertyChanged
+    public abstract class TableUser : INotifyPropertyChanged
```

Comment:
I'm glad this is an abstract class. It's makes the most sense for this class bc it should never be instantiated and this will help enforce that.